ughub.py

```diff
@@ -652,6 +666,33 @@ def GetCurrentRemoteGitURLs(pkg, origin="origin"):
 
     return originFetchURL, originPushURL
 
+def UninstallPackage(args):
+    """
+    Uninstalls all specified packages by parameter args
+    :param args:
+    :return:
+    """
+    packageNames = args
+    options = []
+
+    for i in range(len(args)):
+        if args[i][0] == "-":
+            packageNames = args[0:i]
+            options = args[i:]
+            break
+
+    if len(packageNames) == 0:
+        print("Please specify a package name. See 'ughub help uninstall'.")
+        return
+
+    all_packages = LoadPackageDescs()
+    for package in packageNames:
+        for p in all_packages:
+            if p["name"] == package:
+                if PackageIsInstalled(p):
+                     import shutil
+                     shutil.rmtree(os.path.join(GetRootDirectory(), p["name"]))
```

Comment:
I think this won't work since, e.g., 'plugin' or 'apps' paths are not considered.